core_apps/acme/templates/services/app/cm_live.yml.j2

```diff
@@ -0,0 +1,11 @@
+kind: ConfigMap
```

Comment:
I don't understand how these config maos are used ?

Reply:
If I understand correctly... the `openshift-acme` controller get these ConfigMap by matching their labels :

 - https://github.com/tnozicka/openshift-acme/blob/8b54945f9457b1b960fea133bce359679eee5d62/pkg/api/internal.go#L22-L25
- https://github.com/tnozicka/openshift-acme/blob/8b54945f9457b1b960fea133bce359679eee5d62/pkg/controllerutils/controllerutils.go#L35

And they define the available certificate issuers.

Reply:
![](https://media2.giphy.com/media/tBb19fgUbbUVCNji0iA/giphy.gif)

Reply:
You'll have to checkout the `openshift-acme` code to understand this part.

Reply:
(I didn't see @madmatah comment before answering)